randovania/data/json_data/prime1.txt

```diff
@@ -405,8 +417,16 @@ Asset id: 3006924320
 
 > Pickup (Power Bomb); Heals? False
   * Pickup 38; Major Location? False
+  > Middle-Left Rooftop
+      Trivial
```

Comment:
Leaving from the PB seems wrong. You either need Space Jump or a Bomb Jump to escape the pit

Reply:
Good catch. I couldn't manage a DBJ out, but I also might just be bad at the game.